### ISSUE TITLE
[Feature request] Outline or a tree side panel of the current file.

### ISSUE THREAD

**Issue by astrochili** [inkle/inky#368]
When there are many files and nodes in a script, it becomes necessary to navigate between them and understand the overall structure.

**Comment by olivermarshall2** [comment]
Yes yes and yes. Any sense of being able to visualise the structure, or easily find your way around the structure would be appreciated.

**Comment by lunarcloud** [comment]
Yeah, like software dev tools show you functions in a file - it'd be cool to see the tiered list of knots and stitches in a side panel.

**Comment by lunarcloud** [comment]
Duplicate of #23 and #377
